.github/fabricbot.json

```diff
@@ -1575,6 +1575,488 @@
+            "hours": [
+              2,
+              6,
+              10,
+              14,
+              18,
+              22
+            ],
+            "timezoneOffset": -7
+          },
+          {
+            "weekDay": 1,
+            "hours": [
+              2,
+              6,
+              10,
+              14,
+              18,
+              22
+            ],
+            "timezoneOffset": -7
+          },
+          {
+            "weekDay": 2,
+            "hours": [
+              2,
+              6,
+              10,
+              14,
+              18,
+              22
+            ],
+            "timezoneOffset": -7
+          },
+          {
+            "weekDay": 3,
+            "hours": [
+              2,
+              6,
+              10,
+              14,
+              18,
+              22
+            ],
+            "timezoneOffset": -7
+          },
+          {
+            "weekDay": 4,
+            "hours": [
+              2,
+              6,
+              10,
+              14,
+              18,
+              22
+            ],
+            "timezoneOffset": -7
+          },
+          {
+            "weekDay": 5,
+            "hours": [
+              2,
+              6,
+              10,
+              14,
+              18,
+              22
+            ],
+            "timezoneOffset": -7
+          },
+          {
+            "weekDay": 6,
+            "hours": [
+              2,
+              6,
+              10,
+              14,
+              18,
+              22
+            ],
+            "timezoneOffset": -7
+          }
+        ],
+        "searchTerms": [
+          {
+            "name": "isOpen",
+            "parameters": {}
+          },
+          {
+            "name": "isIssue",
+            "parameters": {}
+          },
+          {
+            "name": "hasLabel",
+            "parameters": {
+              "label": "Icebox cleanup candidate"
+            }
+          },
+          {
+            "name": "noLabel",
+            "parameters": {
+              "label": "Triage:NeedsTriageDiscussion"
+            }
+          },
+          {
+            "name": "noActivitySince",
+            "parameters": {
+              "days": 14
+            }
+          },
+          {
+            "name": "hasLabel",
+            "parameters": {
+              "label": "internal-debugging"
+            }
+          }
+        ],
+        "taskName": "[Close stale icebox][4-2] Close \"Icebox cleanup candidate\" if no activity within 14 days.",
+        "actions": [
+          {
+            "name": "addReply",
+            "parameters": {
+              "comment": "This issue will now be closed since it had been marked Icebox cleanup candidate, but received no further activity in the past 14 days. It is still possible to reopen or comment on the issue, but please note that the issue will be locked if it remains inactive for another 30 days."
```

Comment:
Do we need  the long explanation?

I feel like we already add a few verbose messages, so we can just close it :)

Reply:
Thanks for bring that out! I asked the question in the teams and will update here later.